CompanyEmployees.Core.Domain/Entities/Attendance.cs

```diff
@@ -0,0 +1,29 @@
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace CompanyEmployees.Core.Domain.Entities
+{
+    public class Attendance
+    {
+        [Column("AttendanceId")]
+        public Guid Id { get; set; }
+
+        [Required]
+        [ForeignKey(nameof(Employee))]
+        public Guid EmployeeId { get; set; }
+
+        [Required]
+        public DateTime CheckInTime { get; set; }
+
+        public DateTime? CheckOutTime { get; set; }
+
```

Comment:
Consider making the `Status` property an enum instead of a free-form string to restrict values to a known set (e.g., Present, Absent, Late). This will improve type safety and reduce runtime issues.

**Suggested change**
```suggestion
// Define an enum in a separate file
public enum AttendanceStatus { Present, Absent, Late }

// Then update the property
g[Required]
public AttendanceStatus Status { get; set; }
```